win_etw_macros/src/lib.rs

```diff
@@ -502,6 +534,11 @@ fn trace_logging_events_core(attr: TokenStream, item_tokens: TokenStream) -> Tok
         });
     }
 
+    if errors.len() > 0 {
+        output.extend(errors.into_iter().map(|e| e.to_compile_error()));
+        return output;
+    }
```

Comment:
Please restore the old behavior. We intentionally report errors and still generate code, so that the error debugging experience is as good as possible, for users. Also, in places where you've used `break` to interrupt control flow, please don't break. Just report an error, and continue processing with some sensible default. The Rust compiler will still fail the build, and the user will get the most (best) error detail possible.

Thanks.